.github/workflows/lint.yml

```diff
@@ -130,3 +129,31 @@ jobs:
       - name: Linting
         working-directory: ./
         run: npm run prettier:all
+
+  # 4. YAML
+  yaml:
+    name: YAML 🎨
+    environment:
+      name: ${{ needs.setup.outputs.environment }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Timezone
+        uses: szenius/set-timezone@v1.2
+        with:
+          timezoneLinux: ${{ needs.setup.outputs.timezone }}
+
+      - name: Repository
+        uses: actions/checkout@v4
+
+      - name: Node
```

Comment:
We probably don't need to npm i here as we're using `npx`? to run yaml-lint?  Maybe we could have a locally runable npm script for this for devs to use?

Reply:
Good shout, have added as `validate:yml` script.